### PR TITLE
CRM-19445: Allow to search on membership ID

### PR DESCRIPTION
**Pull request by liedekef**
The member search didn't had an option to quickly search on ID, this should fix that.

---
- [CRM-19445: Allow to search on membership id](https://issues.civicrm.org/jira/browse/CRM-19445)

### CODE REVIEW

File: templates/CRM/Member/Form/Search/Common.tpl [hunk 1]
```diff
@@ -65,6 +65,9 @@
   </td>
 </tr>
 
+<tr><td>{$form.membership_id.label} {$form.membership_id.html}</td>
```
Comment (seamuslee001) [inline templates/CRM/Member/Form/Search/Common.tpl]: @liedekef Franky can you add the <label> tag around {$form.membership_id.label}

Comment (liedekef) [inline templates/CRM/Member/Form/Search/Common.tpl]: Ow sorry, forgot those. Added.

File: CRM/Member/BAO/Query.php [hunk 2]
```diff
@@ -473,6 +473,7 @@ public static function buildSearchForm(&$form) {
     ));
 
     $form->addElement('text', 'member_source', ts('Source'));
+    $form->addElement('text', 'membership_id', ts('Membership ID'));
```
Comment (colemanw) [inline CRM/Member/BAO/Query.php]: I would do `$form->add('number', 'membership_id', ts('Membership ID'), array('class' => 'four', 'min' => 1));

Comment (liedekef) [inline CRM/Member/BAO/Query.php]: Done (although it seems that it still allows you to enter text, and the up/down arrows don't really add extra functionality imho)

**Comment by monishdeb** [comment]
@liedekef thanks for your work, this is really a cool addon to Membership Search form. Is there any jira ticket for this OR can you please raise a ticket (here https://issues.civicrm.org/jira) for this PR ?

**Comment by liedekef** [comment]
@monishdeb : of course: CRM-19445 was created for this.

**Comment by monishdeb** [comment]
@liedekef oh cool, I've edited the PR title. Usually the title contain two basic information which is in ```[Jira ticket number] : [issue title]``` format.  This help reviewer and civicrm-bot to track the related jira ticket.

**Comment by monishdeb** [comment]
Works fine 
image

**Pull request merged**